src/union_room.c

```diff
@@ -3377,6 +3377,9 @@ static void Task_InitUnionRoom(u8 taskId)
 
 bool16 BufferUnionRoomPlayerName(void)
 {
+    if (FlagGet(FLAG_NURSE_UNION_ROOM_REMINDER))
+        return FALSE;
```

Comment:
Is this change necessary?

Reply:
See https://github.com/rh-hideout/pokeemerald-expansion/pull/5050#discussion_r1694276627